Makefile

```diff
@@ -1,6 +1,6 @@
 include private.mk
 
-PARENT_IMG ?= quay.io/freeipa/freeipa-server:fedora-34
+PARENT_IMG ?= $(shell grep -v "^#" PARENT_IMG)
```

Comment:
This code is confusing. It took me a while to realize that you have also a file called ``PARENT_IMAGE``.

Reply:
maybe rename to ``parent_img.conf`` ?

Reply:
Thanks! I will rename the file PARENT_IMG to BASE_IMAGE and I will add some comment before the line for clarification.

Reply:
* add ``ci/config/env`` with shell includes. Comments and empty lines are automatically ignored
* Add ``PARENT_IMG=quay.io/freeipa/freeipa-server@sha256:ba7a482d59a0879bf427ebf8a870cc7f06e6f680c056995933fcbe48b19db0fb`` to the env files
* ``PARENT_IMG ?= $(shell . ci/config/env && echo $PARENT_IMG)``

Reply:
Thanks! updated!